packages/docusaurus-types/src/index.d.ts

```diff
@@ -189,3 +192,20 @@ interface HtmlTagObject {
    */
   innerHTML?: string;
 }
+
+export type ValidationResult<T> = InferType<T>;
```

Comment:
What does this InferType do? wasn't able to find much info about it

Reply:
it create a typescript type from schema